datahub-web-react/src/app/ingest/source/builder/RecipeForm/dbt_cloud.tsx

```diff
@@ -0,0 +1,296 @@
+    name: 'token',
+    label: 'API Token',
+    tooltip:
+        'A service account API token for extracting metadata from dbt Cloud APIs. This token must have the privileges required to read metadata (Metadata permission set).',
+    type: FieldType.SECRET,
+    fieldPath: 'source.config.token',
+    placeholder: 'dbts_ndg_m5oCuSRRC80tpx4ysYfN2tOreiHuATAu5VFcdrkIznQgl4VCOs6w==',
+    required: true,
+    rules: null,
+};
+
+export const DBT_CLOUD_ACCOUNT_ID: RecipeField = {
+    name: 'account_id',
+    label: 'Account ID',
+    tooltip: (
+        <span>
+            <TipSection>The ID of the dbt Cloud account to extract metadata for.</TipSection>
+            <TipSection>
+                This can be found in the URL of your dbt instance: https://cloud.getdbt.com/#/accounts/ACCOUNT_ID/.
+            </TipSection>
+        </span>
+    ),
+    type: FieldType.TEXT,
+    fieldPath: 'source.config.account_id',
+    placeholder: '123',
+    required: true,
+    rules: null,
+};
+
+export const DBT_CLOUD_PROJECT_ID: RecipeField = {
+    name: 'project_id',
+    label: 'Project ID',
+    tooltip: (
+        <span>
+            <TipSection>The ID of the dbt Cloud project to extract metadata for.</TipSection>
+            <TipSection>
+                This can be found in the URL of your dbt instance:
+                https://cloud.getdbt.com/#/accounts/123/projects/PROJECT_ID.
+            </TipSection>
+        </span>
+    ),
+    type: FieldType.TEXT,
+    fieldPath: 'source.config.project_id',
+    placeholder: '456',
+    required: true,
+    rules: null,
+};
+
+export const DBT_CLOUD_JOB_ID: RecipeField = {
+    name: 'job_id',
+    label: 'Job ID',
+    tooltip: (
+        <span>
+            <TipSection>
+                The ID of the dbt Cloud job to extract metadata for. Choose the job that serves as the primary mechanism
+                for updating your production data.
+            </TipSection>
+            <TipSection>The Job ID can be found in the URL on the Jobs tab of dbt Cloud.</TipSection>
+            <TipSection>
+                Ensure that your job enables documentation generation on each run by enabling &apos;Generate Docs&apos;
+                on dbt Cloud.
+            </TipSection>
+        </span>
+    ),
+    type: FieldType.TEXT,
+    fieldPath: 'source.config.job_id',
+    placeholder: '789',
+    required: true,
+    rules: null,
+};
+
+const includeModelsPath = 'source.config.entities_enabled.models';
+export const INCLUDE_MODELS: RecipeField = {
+    name: 'entities_enabled.models',
+    label: 'Include Models',
+    tooltip: 'Whether to include extraction of Models or not.',
+    type: FieldType.BOOLEAN,
+    fieldPath: includeModelsPath,
+    required: false,
+    rules: null,
+    getValueFromRecipeOverride: (recipe: any) => {
+        const includeModels = get(recipe, includeModelsPath);
+        if (includeModels === 'YES' || includeModels === undefined || includeModels === null) {
```

Comment:
probably can be simplified to `if (!includeModels || includeModels === 'YES')`

Reply:
+1